Sources_v3/Controllers/DataController.swift

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// The base class for all controllers. Not meant to be used directly.
+public class DataController: Controller {
+    /// Describes the possible states of `DataController`
+    public enum State: Equatable {
+        /// The controller is inactive. Call `startUpdating` to start listening for changes.
+        case inactive
```

Comment:
Do we need this `state` btw? It seems that the initial state is `localDataFetched` now.

Reply:
After discussion: `inactive` state is `initialized` now.